composer.json

```diff
@@ -13,14 +13,14 @@
   "require": {
     "php": "^7.1",
     "paillechat/php-enum": "^1.2 || ^2.1",
-    "symfony/http-kernel": "^3.4 || ^4.0",
-    "symfony/dependency-injection": "^3.4 || ^4.0",
-    "symfony/config": "^3.4 || ^4.0",
-    "doctrine/dbal": "^2.4"
+    "symfony/http-kernel": "^3.4 || ^4.0 || ^5.0",
+    "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0",
+    "symfony/config": "^3.4 || ^4.0 || ^5.0",
+    "doctrine/dbal": ">=2.4"
```

Comment:
this change allows not only dbal 2.10, but also any of 3.x if released in future. was it intentional?

Reply:
No, it was not my intention as I can't assume which changes will come in dbal3. 
What is your suggestion? 
Strongly define 2.4 || 2.10, or reduce a range using interval like >=2.4 <3?

Reply:
`^2.4` was already permitting 2.10 to come

https://jubianchi.github.io/%5E2.4/2.10.0